mne/epochs.py

```diff
@@ -1003,12 +1003,14 @@ def plot_drop_log(self, threshold=0, n_max_plot=20, subject='Unknown',
     def plot_image(self, picks=None, sigma=0., vmin=None,
                    vmax=None, colorbar=True, order=None, show=True,
                    units=None, scalings=None, cmap='RdBu_r',
-                   fig=None, axes=None, overlay_times=None):
+                   fig=None, axes=None, overlay_times=None,
+                   combine=None, groupby=None):
```

Comment:
This could also get 1 line shorter :)
```python
    def plot_image(self, picks=None, sigma=0., vmin=None, vmax=None, 
                   colorbar=True, order=None, show=True, units=None, 
                   scalings=None, cmap='RdBu_r', fig=None, axes=None, 
                   overlay_times=None, combine=None, groupby=None)
```